core/tests/test_utils_test.py

```diff
@@ -291,24 +289,6 @@ def test_cannot_get_response_with_non_dict_params(self):
             self.get_response_without_checking_for_errors(
                 'random_url', [200], params='invalid_params')
 
-    def test_fetch_gravatar_with_headers(self):
```

Comment:
I believe this test was not working as intended. The code for decoding the gravatar response does not work for JSON, it explicitly asks for a PNG response. Because of this, I've deleted it.

FYI @vojtechjelinek

Reply:
Yep, makes sense to delete it.